python/lsst/cp/pipe/runEotestTask.py

```diff
@@ -179,7 +179,7 @@ def validate(self):
             getattr(self, task).output_dir = self.eotestOutputPath
 
 
-class CpTask(pipeBase.CmdLineTask):
+class RunEotestTask(pipeBase.CmdLineTask):
     """
     Calibration (Products) Production (CP) task.
```

Comment:
Should this topline description be updated too?

Reply:
Yes, yes it should.

Reply:
And, come to think of it, a little more of that docstring too actually...